tests/table_api_tests.py

```diff
@@ -0,0 +1,220 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Unit tests for Superset"""
+import json
+
+from superset import db
+from superset.connectors.sqla.models import SqlaTable
+
+from .base_tests import SupersetTestCase
+
+
+class TableApiTests(SupersetTestCase):
+    resource_name = "table"
+
+    def __init__(self, *args, **kwargs):
+        super(TableApiTests, self).__init__(*args, **kwargs)
+
+    def _get_table(self, user="admin"):
```

Comment:
let's keep get_table functionality here but move assertion to the test methods 
just retun data here and assert on it's length in the test

Reply:
Fixed